homeassistant/config.py

```diff
@@ -444,14 +444,24 @@ def _format_config_error(ex: vol.Invalid, domain: str, config: Dict) -> str:
             auth_conf = [
                 {'type': 'homeassistant'}
             ]
+            _LOGGER.info('A Homeassistant auth provider is automatic'
```

Comment:
The default config with no special cases should not print logging. So let's remove this.